superdesk/default_settings.py

```diff
@@ -638,7 +638,7 @@ def local_to_utc_hour(hour):
 #:
 #: .. versionadded:: 1.32.2
 #:
-FILE_INGEST_OLD_CONTENT_MINUTES = 10
```

Comment:
would be good to raise some error if old config is set and not the new one

Reply:
actually new one will be always set, so just when old one is set and is different from new config value